README.md

```diff
@@ -1,8 +1,56 @@
-# PawnHub
+# Landing page - Remote Chess Experience
```

Comment:
Can you make clear that this project contains not only the landing page but also the web client for the project? Thanks 💙